.github/workflows/integration-test-dumpling-common.yml

```diff
@@ -28,6 +28,13 @@ jobs:
     steps:
       - name: 'checkout repository'
         uses: actions/checkout@v3
+        with:
+          # FIXME: the version of dumpling is getting from git
+          # if it doesn't contain a meaningful git history,
+          # the release version will be empty and the program will panic.
+          #
+          # See the `DUMPLING_LDFLAGS` in `Makefile.common` for more information
+          fetch-depth: 0
```

Comment:
@hawkingrei PTAL, I let it download the full history of TiDB to get the version through `git describe --tags --dirty='-dev'`. I think it's necessary to pass the dumpling integration test :man_facepalming:

Reply:
@lance6716 PTAL

Reply:
I created an issue just a minutes ago. https://github.com/pingcap/tidb/issues/37628

LGTM